src/main/scala/latis/service/hapi/DataService.scala

```diff
@@ -55,46 +130,12 @@ class DataService[F[_]: Concurrent](
               _ => Status.`1410`, _.header
             ).toEither
             fmt       <- _fmt.getOrElse(Csv.validNel).bimap(
-              _ => Status.`1409`, _.format
+              _ => Status.`1409`, f => f
             ).toEither
```

Comment:
Just for some background:

Something like `Either` can be given a `Functor` instance by fixing the left side to a specific type. (This is because `Either` has two "holes" but `Functor` only wants one. We'd say that `Either` has the wrong [kind](https://en.wikipedia.org/wiki/Kind_(type_theory)).) So `Either[A, *]` (which means the left side is always `A` but the right side can vary) can be given a `Functor` instance, but `Either` alone cannot. (This is also why `map` and `flatMap` do stuff to the right side and not the left.)

However, there's such a thing as a [`Bifunctor`](https://typelevel.org/cats/typeclasses/bifunctor.html), which (as the name suggests) does want two holes, so `Either` alone works fine. That's where `bimap` comes from. You can recover `map` from `bimap` by leaving the left side alone, and you get `leftMap` from `bimap` by leaving the right side alone.